tapdb/sqlc/migrations/000028_asset_meta_tlv_fields.up.sql

```diff
@@ -0,0 +1,12 @@
+-- We add a set of new minting/universe related columns. Those will all be
+-- added to the meta record as odd (optional) TLV fields. Therefore, they will
+-- not be set on older assets, which is why we need to make them nullable here.
+ALTER TABLE assets_meta ADD COLUMN meta_decimal_display INTEGER;
+
+ALTER TABLE assets_meta ADD COLUMN meta_universe_commitments BOOL;
+
+-- If there's more than one URL, we're going to join them using the 0x00 control
+-- character (which is invalid in a URL itself).
+ALTER TABLE assets_meta ADD COLUMN meta_canonical_universes BLOB;
```

Comment:
Similar to comment below, should we also restrict this to the the max size, or we'll rely on the RPC parsing/validation logic for that?

Reply:
We do have the parsing logic that makes sure there can't be more than 16 URLs and that each URL isn't longer than 255 characters. But I still added the length limit here, makes sense to me.